README.md

```diff
@@ -1,2 +1,99 @@
 # recipe-app
 A meal finder app that pulls from TheMealDB API to allow users to search for, view recipes, and comment on recipes.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# recipe-app
+
+> A meal finder app that pulls from TheMealDB API to allow users to search for, view recipes, and comment on recipes.
+
+![screenshot](./sec.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML, JavaScript
+- Stylelint, Web Hint, Lighthouse
+- Eslint, Prettier
+
+## Video Demo
+
+[Video Demo](https://www.loom.com/share/65a6d8791efa4f06a6dba4cb9848e4b2)
+## Live Demo
+
+[Live Demo Link](https://recipeappmicroverse.netlify.app/)
+
+## Getting Started
+To get a local copy up and running, follow the steps in Setup below.
+
+### Prerequisites
+Basic knowledge of HTML, CSS, and JavaScript.
+
+### Setup
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+[Download the ZIP from this location](https://github.com/topeogunleye/recipe-app/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:
+
+```bash
+git clone https://github.com/topeogunleye/recipe-app.git
+```
+
+- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up
+
+## 🛠 Installation & Set Up
+
+1. Install Node Modules
+
+   ```sh
+   npm install
+   ```
+
+2. Start the development server
+
+   ```sh
+   npm start
+   ```
+
+3. Open [http://localhost:8080](http://localhost:8080) to view it in the browser.
+
+## 🚀 Building and Running for Production
+
+1. Generate a full static production build
+
+   ```sh
+   npm run build
+   ```
+
```

Comment:
- [ ] Kindly add the steps on how to run your tests to your README file. This way, anyone who clones your project will be able to use it smoothly.